packages/terra-clinical-detail-view/src/DetailListItem.scss

```diff
@@ -0,0 +1,7 @@
+.terraClinical-DetailListItem {
+  min-width: 50%;
+
+  padding-bottom: 0.714rem;
+  padding-right: 0.714rem;
```

Comment:
For padding left and right, you might consider the terra-padding mixin in the case of bidi.